### DISCUSSION TITLE
Setting up scheduled events via the API

### DISCUSSION THREAD

**Discussion by gambina1557** [AxisCommunications/discussions#166] [VAPIX]
Hi, 

I'm trying to set up schedules and rules via the API. Cannot really find anything usable from the documentation for this. Anyone done this before?

The idea would be to split a day into fixed-length segments, and record a video while the rule is active. Currently I'm doing this manually, but it's pretty tedious when there are multiple cameras to be set up. All other configuration already happens via the API.

**Comment by mattias-kindborg-at-work** [comment]
Does [Event and action services](https://www.axis.com/vapix-library/subjects/t10175981/section/t10008227/display) contain what you are looking for?

**Comment by vivekatoffice** [comment]
Related link:
[224](https://github.com/orgs/AxisCommunications/discussions/224)
